src/app/handlers/counting.handler.ts

```diff
@@ -41,4 +55,19 @@ export class CountingHandler implements IHandler {
 
     return isNextNumber && isOnlyNumber;
   }
+
+  private _getEmojis() {
+    // The weird chars 🇫 && 🇧 are converted to the letter emojis in discord
+    this._fizzEmoji =
```

Comment:
You could make a getEmojiOrDefault and call it twice instead of the copypasta

Reply:
I feel like this pattern is in the code a lot. I going to make a getEmoji under guildService